Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostBase.cs

```diff
@@ -230,6 +238,16 @@ protected virtual void SetContent()
             }
         }
 
+        /// <summary>
+        /// Disposes the current instance in response to the parent window getting destroyed.
+        /// </summary>
+        /// <param name="sender">Paramter sender is ignored</param>
+        /// <param name="e">Parameter args is ignored</param>
+        private void OnParentClosed(object sender, EventArgs e)
+        {
+            this.Dispose();
```

Comment:
Is there a reason for `this.Dispose()` vs `Dispose(true)` like done in the DestroyWindowCore method below?  Does `DestroyWindowCore` not get called when the window is closed by the user or something?